src/org/usfirst/frc/team2879/robot/subsystems/DriveTrain.java

```diff
@@ -61,39 +61,37 @@ public void initTalonsConfig() {
 	public void initDefaultCommand() {
 		// Set the default command for a subsystem here.
 		// setDefaultCommand(new MySpecialCommand());
-		setDefaultCommand(new DriveMecanumStick());
+		setDefaultCommand(new DriveMecanumStick(1));
 	}
 	
  	public void drive(double x, double y, double rotation) {
-        drivetrain.driveCartesian( x, y, rotation,0.0);
+        drivetrain.driveCartesian( x, -y, rotation,0.0);
 	}
 	
-	public void drive(Joystick joy) {
-        drive(Robot.oi.getStickX(), Robot.oi.getStickY(),Robot.oi.getStickTwist());
+ 	public void drivePolar(double magnitude, double angle, double rotation) {
+        drivetrain.drivePolar( magnitude, angle, rotation);
 	}
-
-	
-	public void setMotorSpeeds(double xIn, double yIn, double rotation, double driveScale, boolean fieldCentric) {
-		double x;
-		double y;
-				
-		x = xIn;
-		y = yIn;
-		
-		talons[0].set((x + y + rotation));
-		talons[2].set(-(-x + y - rotation));
-		talons[1].set((-x + y + rotation));
-		talons[3].set(-(x + y - rotation));
+ 	
+	public void drive(Joystick joy,double speed) {
+        drive(speed*Robot.oi.getStickX(), speed*Robot.oi.getStickY(), speed*Robot.oi.getStickTwist());
 	}
-	
+
 	public AHRS getNavX() {
 		return navX;
 	}
-
 	
 	public void setNavX(AHRS navX) {
 		this.navX = navX;
 	}
 
-	
+	public void changeBrakeMode(boolean brake) {
```

Comment:
For this method, it would be more meaningful to set up some enumerated constants for BRAKE vs COAST, so that people reading calls to this won't have to think too hard about which mode it is being put into.

Reply:
Wait is there a coasting mode that just stops the motors but doesn’t brake? Otherwise a simple Boolean would be easier for me personally.

Reply:
there is brake mode which is like a brake, and a coast which is like putting it in neutral.

Reply:
I also re-named it setBrakeMode(boolean). true would make it brake mode and false would make it coast.

Reply:
@rfavour sounds like a reasonable compromise.

Reply:
agreed, wrapping up documentation tonight, so should be much more readable for tommorow.